src/aws.ts

```diff
@@ -400,7 +400,7 @@ export namespace $AWS {
      * @see https://docs.aws.amazon.com/lambda/latest/dg/API_Invoke.html
      */
     export const Invoke = makeIntegration<
-      "Lambda.Invoke",
+      "$AWS.Lambda.Invoke",
```

Comment:
This seems like an unrelated change?

Reply:
Just making them consistent. The values were not used before.

Reply:
Wanted them to match to make this part cleaner

![image](https://user-images.githubusercontent.com/289213/178220212-7603d765-a024-4ff0-aad9-d45c15d6f9de.png)